src/mapml/features/featureGroup.js

```diff
@@ -19,8 +19,11 @@ export var FeatureGroup = L.FeatureGroup.extend({
       if(layers.length === 1 && firstLayer.options.link) this.options.link = firstLayer.options.link;
       if(this.options.link){
         M.Feature.prototype.attachLinkHandler.call(this, this.options.group, this.options.link, this.options._leafletLayer);
+        this.options.group.setAttribute('role', 'link');
       } else {
         this.options.group.setAttribute("aria-expanded", "false");
+        // fix issue https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/423
```

Comment:
I think we should remove this comment, I'm not sure it holds true since this is no longer `role="application"` (instead relies on the entire map to be `application` (I also noted this in https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/471#discussion_r636193258).

Reply:
How about linking to https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/471#discussion_r636193258, or maybe I'm misunderstanding?

Reply:
I mean https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/423#issuecomment-824951896, sorry

Reply:
That comment talks about Leaflet's `<img tabindex="0">` markers not being operable without `role="button"`, but for some reason that's not an issue with `<g tabindex="0">` . The main reason we're using `role="button"` is to convey that this component operates as such. And potentially overriding any announcements such as <q>Graphic</q> for `<g>` (we know that's true for `<svg>`, don't remember the case for `<g>`). I don't think we need to link to anything there actually.